s3/pkg/datastore/azure/azure.go

```diff
@@ -377,33 +380,25 @@ func (ad *AzureAdapter) ListParts(ctx context.Context, multipartUpload *pb.ListP
 }
 
 func (ad *AzureAdapter) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3) error {
-
-	object := &pb.Object{
-		BucketName: backendDetail.BucketName,
-		ObjectKey:  "emptyContainer/",
+	randId := uuid.NewV4().String()
+	input := &pb.Bucket{
+		Name: "sample" + randId,
 	}
 
-	bs := []byte{0}
-	stream := bytes.NewReader(bs)
-
-	_, err := ad.Put(ctx, stream, object)
-
+	err := ad.BucketCreate(ctx, input)
 	if err != nil {
-		log.Debug("failed to put object[Azure Blob]:", err)
+		log.Error("Failed to create sample bucket :", err)
```

Comment:
nit: error logs should start with small character as per convention

Reply:
updated